kitchen/README.md

```diff
@@ -28,3 +28,26 @@ $ curl -v -XPOST -H "x-kitchen-cookies: a=b,c=d" $(hostname):PORT
 < Set-Cookie: c=d
 ...
 ```
+
+# Demos
+
+## Image Search and Tagging
```

Comment:
Can we put this content under /demos/tensorflow-image-tagging instead of under /kitchen?

Reply:
Done.